testsuite/tests/parallel/domain_parallel_spawn_burn.ml

```diff
@@ -3,6 +3,7 @@
 include unix
 ** bytecode
```

Comment:
Perhaps you may need a separate `ocamlrunparam` for bytecode too? @shindere.

Reply:
locally, the test case passes 20 times in a row

Reply:
Welcome to multicore hacking :-) The github actions VMs have quite good at smoking out non-determinism bugs. I still don't know whether the `ocamlrunparam` as written applies only to the native runs or both of them.